src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -145,19 +144,14 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
 
   // @debt remove backButton from Navbar
   return (
-    <RelatedVenuesProvider venueId={venue.id}>
-      <ReactionsProvider venueId={venue.id}>
-        <WithNavigationBar
-          fullscreen={fullscreen}
-          hasBackButton={hasBackButton}
-        >
-          <AnnouncementMessage message={venue.bannerMessage} />
-          {template}
-          <ChatSidebar venue={venue} />
-          <UserProfileModal venue={venue} />
-        </WithNavigationBar>
-      </ReactionsProvider>
-    </RelatedVenuesProvider>
+    <ReactionsProvider venueId={venue.id}>
+      <WithNavigationBar fullscreen={fullscreen} hasBackButton={hasBackButton}>
+        <AnnouncementMessage message={venue.bannerMessage} />
+        {template}
+        <ChatSidebar venue={venue} />
+        <UserProfileModal venue={venue} />
+      </WithNavigationBar>
+    </ReactionsProvider>
   );
```

Comment:
Why was `RelatedVenuesProvider` removed from here to move it to `AppRouter`? Feels wrong to me.

Reply:
Superseded by `AdminRouter.tsx`'s `Provided` wrapper in 5f2ba2663620de1631552bd47e3d59a3dcab5ce2

```
          <Route path="/in/:venueId">
            <Provided withWorldUsers withRelatedVenues>
              <VenuePage />
            </Provided>
          </Route>
```